controllers/aero_info_calls.go

```diff
@@ -114,6 +114,11 @@ func (r *SingleClusterReconciler) waitForNodeSafeStopReady(
 		return reconcileRequeueAfter(60)
 	}
 
+	ns, err := r.removedNamespaces()
```

Comment:
Can you rename `ns` to `removedNSes` or something to that effect just to make it clean later on